src/features/users.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+type UsersState = {
+  users: User[];
+};
+
+const initialState: UsersState = {
+  users: [],
+};
+
+export const initUsers = createAsyncThunk('users/fetch', () => {
+  return getUsers();
+});
```

Comment:
Consider adding error handling for the `getUsers` API call to manage potential failures gracefully. This will help in providing feedback to the user if the user fetching fails.